dubbo-common/src/main/java/org/apache/dubbo/common/EnvironmentConstants.java

```diff
@@ -0,0 +1,60 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
```

Comment:
suggest to put all constants in Constants, and make it an interface. After that, let's revisit this class to consider if it is possible to split. What do you think?

Reply:
On it. One quick check, if I change the Constants to an interface and dubbo ecosystem projects(outside of incubator-dubbo) are using this class by extending then they might start failing. Do you know any one of them using it? If no one is using then we can change it to interface now.

Reply:
Leaving the changing of java class to interface change. Or if you feel we should go ahead and make change please let the comment I will make the changes as part of this PR only.